packages/utils/src/useDedicatedAction/useDedicatedAction.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+// TODO Refactor.
+const useDedicatedAction = (options) => {
```

Comment:
This feels more like a helper function rather then hook. Can we omit the `use` from the name of the function?